### ISSUE TITLE
Support unmyelinated axons in morphometrics

### ISSUE THREAD

**Issue by hermancollin** [neuropoly/axondeepseg#795]
**Describe the problem**
I received an email from our Sick Kids collaborators and they are currently collecting morphometrics for unmyelinated axons using ADS. The only problem is that she has to manually combine the myelinated/unmyelinated axon masks + update the axonmyelin mask. She has a Fiji macro for this but it is still impractical because she has to open each image one by one.

**Proposed solution**
The morphometrics script could simply have an option to incorporate the unmyelinated axon mask before the call to `compute_morphometrics()`.

Closely related to #698

**Comment by Stoyan-I-A** [comment]
It is possible to compute the morphometrics with only an axon mask by calling `get_axon_morphometrics` and setting the `im_myelin` to `None`.

**Comment by Stoyan-I-A** [comment]
Would it be sufficient to just add this option to the batch morphometrics? And we would just not pass a myelin mask to the function that computes the morphometrics.

**Comment by hermancollin** [comment]
> It is possible to compute the morphometrics with only an axon mask by calling `get_axon_morphometrics` and setting the `im_myelin` to `None`.

Ideally I think it would be better to call the morphometrics script only once and it would do it for both myelinated and unmyelinated axons but maybe there is a better way.

> Would it be sufficient to just add this option to the batch morphometrics? And we would just not pass a myelin mask to the function that computes the morphometrics.

@Stoyan-I-A Why would we not want it for the normal (non-batched) morphometrics tho?

**Comment by hermancollin** [comment]
> It is possible to compute the morphometrics with only an axon mask by calling get_axon_morphometrics and setting the im_myelin to None.

@Stoyan-I-A actually yes it might be nice to have this ALSO (in case someone wants to apply morphometrics only to an unmyelinated axon mask). Maybe we could automatically detect if there is no myelin mask and in this case proceed like you said.

**Comment by Stoyan-I-A** [comment]
> Why would we not want it for the normal (non-batched) morphometrics tho?

We can add it to both.

I'm just trying to understand what the user wants exactly.

**Comment by Stoyan-I-A** [comment]
> unmyelinated axon mask

Is this going to be like a third class (4th if you include the background)? Does the user data contain masks which are currently labeled with all these masks: _background, myelin, myelinated axons, unmyelinated axons_?

**Comment by hermancollin** [comment]
@Stoyan-I-A at the moment yes because the unmyelinated axon prediction is obtained by a 2nd model.

**Comment by hermancollin** [comment]
I ended up doing something slightly different. I implemented a draft in #796 and it is ready to test.

To simplify things, the `-u` flag toggles unmyelinated mode. In this mode, axon and myelin masks are ignored and the script looks for files suffixed `_seg-uaxon` instead. I used the fact that the `get_axon_morphometrics(...)` function works without a myelin mask.
